.github/actions/height-integration/sync-pr/pr-opened-action.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync new PR to Height
+description: Moves any issue-related tasks in Height into the 'In Progress' swimlane on the interface project board
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move PR-related tasks into 'In Progress' in Height
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.height.app/tasks'
+          method: 'PUT'
+          customHeaders: '{ "Content-Type": "application/json", "Authorization": "api-key ${{ secrets.HEIGHT_SECRET_TOKEN }}" }'
+          # data: '{ "name": "${{ github.event.issue.title }}", "listIds": ["64fa570a-a252-4521-b801-9b0eb18113bc"], "status": "359d42b4-5e7b-46fa-8465-d179b5a097ef" }'
```

Comment:
hmm

Reply:
I think it's just a matter of knowing what list IDs or project board ID we're working with firstly. And really just looking at the API docs for payload for this `/tasks` endpoint